yarpl/include/yarpl/flowable/FlowableOperator.h

```diff
@@ -51,7 +51,8 @@ class FlowableOperator : public Flowable<D> {
 
     void subscriberOnNext(D value) {
       if (subscriber_) {
-        subscriber_->onNext(std::move(value));
+        auto ref = subscriber_;
```

Comment:
curious whether we have a test case covering this case. Do we need this here?
I just feel like we are sprinkling stack ref copies to a bunch of places and I don't know if we we are safe now or still not yet. I also don't know any better solution.

Reply:
You're right, it might not be sufficient to sprinkle these just in the locations where we call methods on the `subscriber_`; we need a better solution.

Reply:
Can't believe I'm saying this but I'm wondering if the best path forward for now is to disable the failing tests (there should only be two in trunk, `RequestStreamTest.OperationsAfterCancel` and `Observable.CancelSubscriptionChain`).

If the issue is more systemic than a few lines of code can fix, maybe we should table any "better" solutions for later.